---
bumperbot_detection/LitterPoint litter
bool success